Scripts/Miscellaneous/Notepad_clone/notepad.py

```diff
@@ -0,0 +1,188 @@
+from PyQt5.QtGui import *
+from PyQt5.QtWidgets import *
+from PyQt5.QtCore import *
+from PyQt5.QtPrintSupport import *
+
+import os
+import sys
+
+
+class MainWindow(QMainWindow):
+
+    def __init__(self, *args, **kwargs):
+        super(MainWindow, self).__init__(*args, **kwargs)
+
+        layout = QVBoxLayout()
+        self.editor = QPlainTextEdit()  # Could also use a QTextEdit and set self.editor.setAcceptRichText(False)
+
+
+        # Setup the QTextEdit editor configuration
+        fixedfont = QFontDatabase.systemFont(QFontDatabase.FixedFont)
+        fixedfont.setPointSize(12)
+        self.editor.setFont(fixedfont)
+
+        # self.path holds the path of the currently open file.
+        # If none, we haven't got a file open yet (or creating new).
+        self.path = None
+
+        layout.addWidget(self.editor)
+
+        container = QWidget()
+        container.setLayout(layout)
+        self.setCentralWidget(container)
+
+        self.status = QStatusBar()
+        self.setStatusBar(self.status)
+
+        file_toolbar = QToolBar("File")
+        file_toolbar.setIconSize(QSize(14, 14))
+        self.addToolBar(file_toolbar)
+        file_menu = self.menuBar().addMenu("&File")
+
+        open_file_action = QAction(QIcon(os.path.join('images', 'blue-folder-open-document.png')), "Open file...", self)
```

Comment:
limit 79 characters in single line
```suggestion
       open_file_action = QAction(QIcon(os.path.join('images',
                           'blue-folder-open-document.png')),
                           'Open file...', self)
```